tests/utils.py

```diff
@@ -4,9 +4,18 @@
 import shutil
 from pathlib import Path
 
+def download_datafiles(pathname, channels, numbers, output_dir, gb_url):
+    paths = [f"{pathname}_{ch}_{n}.ibw" for ch in channels for n in numbers]
+
+    Path(output_dir).mkdir(exist_ok=True, parents=True)
+    for path in paths:
+        output_path = f"{output_dir}{path}"
+        if not Path(output_path).is_file():
+            with urllib.request.urlopen(f"{gb_url}{path}") as response, open(output_path, "wb") as out_file:
+                shutil.copyfileobj(response, out_file)
 
 def download_sahp_datafiles():
```

Comment:
maybe we should change the name of the function in include also idthresh

Reply:
Ah no, idthresh is only used for computing rheobase, we are mainly interested in sahp data.

Alright, no need to change the name of the function, you can ignore the above comment